### PR TITLE
Bah-2794: Added a run-bahmni.sh script for common docker compose commands, for making life easy!

### PR DESCRIPTION
**Pull request by gsluthra**
Added a utility wrapper script for making it easy for new users to use docker compose. This script also checks if docker is installed, and min version of docker. It will only run from subfolders bahmni-lite or bahmni-standard. 

Sample output:

image

### CODE REVIEW

**Comment by gsluthra** [comment]
JIRA issue: https://bahmni.atlassian.net/browse/BAH-2794

File: run-bahmni.sh [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    esac
+}
+
+
+function resetAndEraseALLVolumes {
+  if confirm "Are you sure you want to DELETE all Data and Volumes in docker on this MACHINE?? Please say Yes/No to proceed if you are absolutely sure. This will delete even non-Bahmni volumes used by Docker!"; then
+    echo "Proceeding with a DELETE.... "
+    
+    echo "1. Stopping all services.."
+    docker compose down
```
Comment (mohan-13) [inline run-bahmni.sh:124]: We can use -v flag in  this command to remove volumes created by docker compose.

File: run-bahmni.sh [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    docker compose ps
+    
+    echo "2. Deleting all volumes .."
+    docker volume rm $(docker volume ls -q)
```
Comment (mohan-13) [inline run-bahmni.sh:128]: This command can be dangerous, if the user has volumes that are not related to Bahmni, those would get cleared off as well.

**Review by mohan-13 — CHANGES_REQUESTED** [review]

File: run-bahmni.sh (hunk 1 above)
Comment (gsluthra) [reply run-bahmni.sh:124]: Let me try -v. The problem is, that if a service is not running, its volumes will not be deleted. Would want to delete all BAHMNI related volumes.

Comment (gsluthra) [reply run-bahmni.sh:124]: Done. Please see.

File: run-bahmni.sh (hunk 2 above)
Comment (gsluthra) [reply run-bahmni.sh]: Done please see.

**Review by mohan-13 — APPROVED** [review]
